example/swift/swift_api.h

```diff
@@ -1,4 +1,4 @@
-// Generated by Apple Swift version 5.7.2 (swiftlang-5.7.2.135.5 clang-1400.0.29.51)
+// Generated by Apple Swift version 5.8.1 (swiftlang-5.8.0.124.5 clang-1403.0.22.11.100)
```

Comment:
This should be inside a third_party folder example/swift/third_party/swift_api.h

See https://opensource.google/documentation/reference/thirdparty/non-google3#i_need_to_include_third-party_source_code_in_a_dart_package